cap/modules/theme/templates/cap_theme/header_login.html

```diff
@@ -25,9 +25,9 @@
 
 {% block navbar_login %}
   {%- if not current_user.is_authenticated %}
-  <form class="navbar-right">
-    <a  href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="btn btn-default sign-in-btn btn-lg" >{{_('Sign in with CERN')}}</a>
-  </form>
+  <div>
+    <a id="login-button"  href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="btn btn-primary btn-lg" >{{_('Log in with your CERN account')}}</a>
```

Comment:
```
#login-button {
    background-color: $theme-color;
    padding: 9px 14%;
    margin-top:2%;
    font-size: 16px;
    border-radius: 2px;
}
```
As it is now it is defining the style of the element that has id login button. As we are going  to need the same style as the button  in the homepage you can turn the id to a class like 
```
.login-button {....}
```
and use it in both places. You have to do some modifications though so as the button to be smaller only in the header and remain as it is in the homepage.